src/uwtools/tests/drivers/test_driver.py

```diff
@@ -82,6 +100,153 @@ def schema():
 """.strip()
 
 
+def test_create_directory_structure(tmp_path):
+    run_dir = tmp_path / "run"
+    with patch.object(Driver, "_create_run_directory") as _create_run_directory:
+        Driver.create_directory_structure(run_dir)
+    _create_run_directory.assert_called_once_with(run_dir, "delete")
+
+
+def test_create_directory_structure_bad_existing_act():
+    with raises(ValueError):
+        Driver.create_directory_structure(run_directory="/some/path", exist_act="foo")
+
+
+def test_run_cmd_expected(mpi_config, tmp_path):
+    config_file = tmp_path / "config.yaml"
+    YAMLConfig.dump_dict(config_file, mpi_config)
+    with patch.object(Driver, "_validate", return_value=True):
+        driver = ConcreteDriver(config_file=config_file)
+    driver._config = driver._experiment_config.get("component", {})
```

Comment:
Is there doubt here as to whether `.get()` is going to find `component`? I'm surprised by the use of `.get()` instead of `[]`, and the supply of a default value. Same question about the next test function...